pyproject.toml

```diff
@@ -1,50 +1,57 @@
-[tool.poetry]
+[project]
```

Comment:
switched this over to using a more generic format, which will make the transition to using uv, if such a transition occurs, smoother.